src/Filtering/ImageStatistics/CalculateImageMoments/Documentation.rst

```diff
@@ -0,0 +1,76 @@
+Calculate Image Moments
+================================
+
+.. index::
+   single: ImageMomentsCalculator
+
+Synopsis
+--------
+
+
+Calculate image moments from binary images and an interactive ellipse.
+
+
+Results
+-------
+
+.. figure:: ellipse.mha
+  :scale: 50%
+  :alt: Input image
+
+  Input Image
+::
+Output
+ ImageMomentsCalculator (000002037138BE90)
+   RTTI typeinfo:   class itk::ImageMomentsCalculator<class itk::Image<unsigned short,2> >
+   Reference Count: 1
+   Modified Time: 249
+   Debug: Off
+   Object Name:
+   Observers:
+     none
+   Image: 000002037138FDD0
+   Valid: 1
+   Zeroth Moment about origin: 153
+   First Moment about origin: [30, 50]
+   Second Moment about origin: 15.0458 -8.8366
+ -8.8366 15.0458
+
+   Center of Gravity: [30, 50]
+   Second central moments: 15.0458 -8.8366
+ -8.8366 15.0458
+
+   Principal Moments: [950, 3654]
+   Principal axes: 0.707107 0.707107
+ -0.707107 0.707107
+::
+
+
+Jupyter Notebook
+-----------------
+
+.. image:: https://mybinder.org/badge_logo.svg
+   :target: https://mybinder.org/v2/gh/InsightSoftwareConsortium/ITKExamples/master?filepath=src%2FFiltering%2FImageStatistics%2FImageMomentsCalculator%2FCalculateEllipseMoments.ipynb
+
+Code
+----
+
+C++
```

Comment:
I think we recently switched convention to have Python code before C++.

Reply:
If so the cookiecutter hasn't been updated.

https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/blob/fd2b39b4d652d89e9a7de53ea1c947cd8a597c30/Utilities/CookieCutter/%7B%7Bcookiecutter.example_name%7D%7D/Documentation.rst

Reply:
I guess it should be updated, as it was simply forgotten. Is that right @mseng10?